docker/repository/pom.xml

```diff
@@ -10,18 +10,10 @@
 
     <properties>
         <alfresco.gs.repo.version>3.1.0</alfresco.gs.repo.version>
-        <alfresco.sync.service.version>3.1.0</alfresco.sync.service.version>
```

Comment:
We'll still need to add the Sync AMP until it's part of the default repo image.

Reply:
Sync is part of the default repo image, both ACS 6.1.1 and latest(6.2.0)